frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/health-card/health-card.tsx

```diff
@@ -15,33 +16,59 @@ import {
 import { HealthBody } from '@console/internal/components/dashboard/health-card/health-body';
 import { HealthItem } from '@console/internal/components/dashboard/health-card/health-item';
 import { HealthState } from '@console/internal/components/dashboard/health-card/states';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { FirehoseResource } from '@console/internal/components/utils';
+import { CephClusterModel } from '../../../../models';
+import { CEPH_STORAGE_NAMESPACE } from '../../../../constants/index';
 import { STORAGE_HEALTH_QUERIES, StorageDashboardQuery } from '../../../../constants/queries';
 import { filterCephAlerts } from '../../../../selectors';
 import { getCephHealthState } from './utils';
 
+const cephClusterResource: FirehoseResource = {
```

Comment:
this is still old definition. Can you specify the definition in one place and reuse it everywhere so we will avoid these issues in the future ?

Reply:
Have defined the definition at frontend/packages/ceph-storage-plugin/src/constants/resources.ts 
Forgot to change it here. Thanks for the catch. Will update